cmd/sifnoded/genaccounts.go

```diff
@@ -224,3 +227,61 @@ contain valid denominations. Accounts may optionally be supplied with vesting pa
 
 	return cmd
 }
+
+func AddGenesisValidatorCmd(
+	ctx *server.Context, cdc *codec.Codec, defaultNodeHome, defaultClientHome string,
+) *cobra.Command {
+
+	cmd := &cobra.Command{
+		Use:   "add-genesis-validators [address_or_key_name]",
+		Short: "add genesis validators to genesis.json",
+		Long: `add validator to genesis.json. The provided account must specify
+the account address or key name. If a key name is given, the address will be looked up in the local Keybase. 
+`,
+		Args: cobra.ExactArgs(1),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			config := ctx.Config
+			config.SetRoot(viper.GetString(cli.HomeFlag))
+			fmt.Printf("argument is  %v \n", args[0])
```

Comment:
Need to remove debug printfs

Reply:
removed.

> Could you please add a readme explaining how this validator permissioning works, and how you can add or remove validators as an admin?

Current implementation just set the whitelist validators in genesis, not interface to add/remove from admin account.
@intl-man @utx0  this feature will add one more step to set whitelist validators in genesis. Could you please review the pr to see if any change needed in chianops, and tell me which document file is the best to describe it.